x-pack/plugins/fleet/public/applications/fleet/sections/agent_policy/edit_package_policy_page/index.tsx

```diff
@@ -214,8 +226,32 @@ export const EditPackagePolicyPage: React.FunctionComponent = () => {
     [updatePackagePolicy]
   );
 
-  // Cancel url
-  const cancelUrl = getHref('policy_details', { policyId });
+  // Cancel url + Success redirect Path:
+  //  if `from === 'edit'` then it links back to Policy Details
+  //  if `from === 'package-edit'` then it links back to the Integration Policy List
+  const cancelUrl = useMemo((): string => {
+    if (packageInfo && policyId) {
+      return from === 'package-edit'
```

Comment:
If we expect this to expand to more pages, perhaps a `switch` here so we can be explicit about the default case (if `from` is something unexpected, perhaps new places added down the road, where the PR forgot to add themselves here).

Reply:
yeah, agreed a switch will make sense if we add more types for `from`. I don't see that happening at this time and when I get around to introducing the layout that the mocks call for, I hope to revert this in favor of having props that specifically allow for setting things like the `cancelUrl` that should be used in the form.